app/src/src/screens/menu/EditProfileScreen.jsx

```diff
@@ -1,9 +1,17 @@
 import React, { Component } from 'react';
 import EditProfileView from '../../components/views/EditProfileView';
 import { getUserData } from '../../utils/storageAccess';
-
+// import { JobsEndpoint } from '../../utils/endpoints';
+// import { ActivityIndicator } from 'react-native';
+// Some parts may be used for education/jobs section of edit profile page
 class EditProfileScreen extends Component {
-  state = { user: {} };
+  state = {
+    user: {},
+    /* pages: false,
+    nextPage: 1,
+    jobs: [],
+    done: false, */
+  };
 
   constructor(props) {
     super(props);
```

Comment:
If you're going to set the initial state outside the constructor, then there is no need to write out the constructor.